ci/scripts/connect-release-staging.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+# Usage:
+# ./connect-release.sh DESTINATION
```

Comment:
It looks there is a typo since the script file name is `ci/scripts/connect-release-staging.sh`

Reply:
yeah, this is just a comment, definitely should be resolved but I would first like to land on the final decision. 2 scripts or not?

Reply:
Ok, let's logic about it:
* 1 script is easier to maintain for sure
* as you mentioned there is the risk of human using the script wrongly and deploying to wrong environment

Just brainstorming but what about 3 scripts? one with all the logic (it will make easy to maintain) and 2 for each environment with the variables of the environment.
With this approach it is more difficult for a human to make the mistake to use the wrong script but at the same time you have the logic in only 1 script and that makes easy to maintain.

Reply:
yes, why not, that could work as well. what I am presonally most afraid of is some variable expansion in gitlab/github yaml files goes wrong and this script gets called without variables and/or fallbacks to something we don't expect.

Reply:
Why 3 scripts? It makes no sense, let's keep as simple as possible. 1 script, one purpose that's it. I think the way you've done it here is fine. Human wont have authority to execute the stuff from the script ci is the only way.